src/ott/solvers/linear/sinkhorn.py

```diff
@@ -409,7 +410,8 @@ def converged(self) -> bool:  # noqa: D102
     if self.errors is None:
       return False
     return jnp.logical_and(
-        jnp.any(self.errors == -1), jnp.all(jnp.isfinite(self.errors))
+        jnp.logical_not(jnp.any(jnp.isnan(self.errors))),
+        self.errors[-1] < self.threshold
```

Comment:
Maybe mention in the comments that this relies on the fact that `errors` are initialized with `-1` and `threshold > 0`, so if it converges prematurely, this passes, if not the error is always computed at the last iteration.